src/models/RevAiApiCustomVocabulary.ts

```diff
@@ -0,0 +1,10 @@
+import {CustomVocabularyStatus} from './CustomVocabularyStatus';
+
+export interface RevAiApiCustomVocabulary {
```

Comment:
@kbridbur How should we best distinguish these named properties?

`CustomVocabularyInformation`?

Reply:
Do you mean distinguish them from Job?  I'm slightly confused

Reply:
I'm also not clear on what Andrew means here.

Reply:
@amikofalvy ping

Reply:
Ahh I see what he means now.